app/templates/base.html

```diff
@@ -11,19 +11,19 @@ <h4>resource</h4>
                     <div class="army-resources attack">
                     <div class="user-resource">
                         <span>gold</span>
-                        <span>*number*</span>
+                        <span>{{ current_user.army[0].gold }}</span>

```

Comment:
Isn't `army` not a list anymore? No need to use `[0]`

Reply:
doesn't work, i will try to fix that today